### ISSUE TITLE
Add batch downloading and automatic detection and addition of and automatic paste url addresses of the clipboard without clicking paste button

### ISSUE THREAD

**Issue by trimechee** [StefanLobbenmeier/youtube-dl-gui#118]
Hello, thank you so much fir this amazing great fantastic downloader ! 💯 🥇 

I have request please :  sometimes we need to download quickly specific videos from a YouTube channel, so to speed up and simplify the process and avoid clicking on "paste link" each time, it would be great please if our beloved Youtube-dl-gui could detect the url address contained in the clipboard and automatically add it and paste it to the batch downloader, thank you so much !

**Comment by StefanLobbenmeier** [comment]
Yeah I plan on adding a setting for automatically adding or downloading anything in the clipboard
